sonic_installer/bootloader/uboot.py

```diff
@@ -23,12 +23,12 @@ class UbootBootloader(OnieInstallerBootloader):
 
     def get_installed_images(self):
         images = []
-        proc = subprocess.Popen("/usr/bin/fw_printenv -n sonic_version_1", shell=True, text=True, stdout=subprocess.PIPE)
+        proc = subprocess.Popen(["/usr/bin/fw_printenv", "-n", "sonic_version_1"], text=True, stdout=subprocess.PIPE)
         (out, _) = proc.communicate()
         image = out.rstrip()
         if IMAGE_PREFIX in image:
             images.append(image)
-        proc = subprocess.Popen("/usr/bin/fw_printenv -n sonic_version_2", shell=True, text=True, stdout=subprocess.PIPE)
+        proc = subprocess.Popen(["/usr/bin/fw_printenv", "-n", "sonic_version_2"], text=True, stdout=subprocess.PIPE)
```

Comment:
Should we force user use `run_command` instead of `subprocess.Popen`?

Reply:
we can but it can change original behavior that it will print the running command to the output and exit if the command fails.

https://github.com/sonic-net/sonic-utilities/blob/5e53d83deb955d335a7bce1dbf2c1b4545ada3c8/sonic_installer/common.py#L25

https://github.com/sonic-net/sonic-utilities/blob/5e53d83deb955d335a7bce1dbf2c1b4545ada3c8/sonic_installer/common.py#L36